### DISCUSSION TITLE
Improve map legibility

### DISCUSSION THREAD

**Discussion by Esmolante** [organicmaps/organicmaps#2970] [Ideas]
I think map display is where OM could gain the most. For example, there is no color hierarchization of roads, beyond highways and national roads, and could be very usefull. Also, the map is saturated with place names that are only relevant to the inhabitants of a village, but appear at the same level as the name of that village, generating confusion. A huge improvement for offroad use would be to use isolines at 10m, instead of 50m. And eliminate the striping in natural parks, which greatly limits legibility.

**Comment by Esmolante** [comment]
Let me give you an example. I hope no one will be offended if I use another programme as a term of comparison. It is a Czech browser, Mapy.sz, which for me is the benchmark in terms of map readability.
OM:
![OM](https://esmola.files.wordpress.com/2022/08/screenshot_2022-08-07-22-48-08-593_app.organicmaps.jpg)

Mapy:
![Mapy](https://esmola.files.wordpress.com/2022/08/screenshot_2022-08-07-22-48-51-614_cz.seznam.mapy_.jpg)

The difference is obvious. The map of OM is blurred by a multitude of toponyms in grey letters (place=locality) that are only meaningful to people from that village, and which are not even known or cared about by those in the next village. In the end, it is confusing to distinguish between village names (in black) and place names (in grey).

In addition, we see that streets (highway=residential), roads (highway=secondary) and tracks (highway=unclassified) are shown with the same graphics. But when it comes to visually searching for a road, one is not the same as the other. Roads have a hierarchy, and it is important for the driver to be able to visualise this hierarchy quickly (different colours and line widths).

**Comment by Esmolante** [comment]
Another example. Let's zoom out in another place to see what I said about the importance of road hierarchy.

OM:
![OM](https://esmola.files.wordpress.com/2022/08/screenshot_2022-08-07-23-41-10-085_app.organicmaps.jpg)

Mapy:
![Mapy](https://esmola.files.wordpress.com/2022/08/screenshot_2022-08-07-23-39-54-900_cz.seznam.mapy_.jpg)

And, even better, IGN (Spanish National Geographic Institute):
![IGN](https://esmola.files.wordpress.com/2022/08/screenshot_20220808_000007.png).

Imagine you have never been there before. With which map would you be able to find the best route from one point to another?  Different categories of roads should be represented graphically in a differentiated way.

And another issue. Do you see how visually annoying it is to have the map crossed with stripes to represent a natural park? Just show the perimeter with another ink and that's it.

**Comment by not-available-username** [comment]
Totally agree, some things are mentioned here https://github.com/organicmaps/organicmaps/issues/755, like the strange choice to show caves, waterfalls and small airfield before like anything 😅

**Comment by Esmolante** [comment]
Let me give you another example. Let's take the calculation of this route.

![Screenshot_2022-10-18-14-19-08-306_app organicmaps](https://user-images.githubusercontent.com/109362590/196432415-5d5f80d0-dc1f-47ba-b619-8aa80d1160e7.jpg)

Can you tell me whether is the most convenient route?

![Screenshot_2022-10-18-14-19-28-515_app organicmaps](https://user-images.githubusercontent.com/109362590/196433501-44b47984-23f2-46a2-a9b2-20aad741987f.jpg)

It seems so, all the streets look the same and it is the shortest route.

Now let's see with another app (Mapy.cz)

![Screenshot_2022-10-18-14-20-33-279_cz seznam mapy](https://user-images.githubusercontent.com/109362590/196434475-1ef577c6-9875-44f9-b5ff-783c71dba507.jpg)

By showing the hierarchy of streets, we can see at a glance that there is a road running through the village. It is preferable to follow this road than to look for shortcuts through the narrow streets.

Another graphical detail that could be interesting is to replace the zoom buttons (+ and -) with a slider.

**Comment by patepelo** [comment]
@organicmaps/styles Hey, maybe we should do a thin tank soon?

**Reply by pastk** [reply]
We can have a call and a talk
